### PR TITLE
Add image seeds

### LINKED ISSUE #5: Image Seeds
#### As a developer I want at least 20 images to be created when I setup the database.

Acceptance criteria:
- [x] After running `rake db:setup` at least 20 images are shown in the
  homepage.
- [x] When a new app is deployed on heroku (e.g., every review app), that app
  automatically has at least 20 images on its homepage.

Dependencies:
- Image Index

### PR DESCRIPTION
**Pull request by wpliao1989**
Closes #5 

---

The following is a checklist to remind you of the steps to take when opening a
new PR. The horizontal line above and everything below can be removed from the
pull request description.

- [x] I have replaced `X` above with the associated issue number.
- [x] I have changed the issue's label to "pull request".
- [x] I have assigned the designated reviewer to this PR.
- [x] I have set the appropriate label on this PR.

If needed, please review the document on the
[Image Sharing Project](https://sites.google.com/a/appfolio.com/eng/new-hire-resources/engineering-academy/ropes/image-sharing-project)
for information on how labels should be used.

### CODE REVIEW

**Comment by wpliao1989** [comment]
Hi @mlewisno, could you review this when you have a minute?

File: db/seeds.rb [hunk 1]
```diff
@@ -5,3 +5,24 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+Image.create(url: 'https://images-na.ssl-images-amazon.com/images/G/01/img15/pet-products/small-tiles/30423_pets-products_january-site-flip_3-cathealth_short-tile_592x304._CB286975940_.jpg')
```
Comment (mlewisno) [inline db/seeds.rb]: Hey, two quick thoughts:
1. There's a lot of calls to `Image.create` in here. You can actually pass a hash into the create method using a hash that looks something like this:
```
{
  url: http://blahblah.com,
  url: http://blahblah.com,
  url: http://blahblah.com
}
```
This means you only have to make one call to Image.create, and also if you need to change that call you don't have to do it 20 times.

2. On that note what would happen if the URL was invalid on one of those create calls? How would you know?

Comment (wpliao1989) [inline db/seeds.rb]: 1. Good tip!
2. I would never know, unless I use the images on the server.

**Comment by wpliao1989** [comment]
Updated

**Comment by mlewisno** [comment]
Looks great!

One quick point that isn't worth going back to change - In general at appfolio we put our initials at the beginning of a commit message so that people looking back over the code know who they can talk to about it, fir an instance `wp - Throw exception when seed image creation fails`

**Pull request merged**